app.py

```diff
@@ -0,0 +1,81 @@
+import os
+import requests
+from flask import Flask
+from flask import jsonify
+from flask import request
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+from utils import create_markdown_list
+from utils import generate_randomized_list
+from utils import generate_demo_order_blocks
+from utils import verify
+from utils import verify_is_valid_demo_day
+
+# channel_id = os.environ["SLACK_CHANNEL_ID"]
+slack_token = os.environ["SLACK_OAUTH_TOKEN"]
+sc = WebClient(token=slack_token)
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def home():
+    return "Homepage"
+
+
+@app.route("/set-start-date", methods=["POST"])
+def set_start_time():
+    # if no start time found
+    # prompt for start date
+    # calculate all valid demo days starting from 14 days from start date
+    # store list of valid demo days in db
+    # else if there is a start time
+    # confirm overwrite
+    pass
+
+
+@app.route("/get-scheduled-dates", methods=["GET", "POST"])
+def get_scheduled_dates():
+    # if there is a stored list of dates
+    # get current date
+    # return list of valid demo days after the current date
+    # else if no dates stored
+    # return no scheduled dates msg
+    pass
+
+
+@app.route("/delete-schedule", methods=["POST"])
+def delete_schedule():
+    # if there are valid demo days
+    # delete stored valid demo days
+    # else if no dates stored then no action is taken
+    # alert nothing was done
+    pass
+
+
+@app.route("/slash-get-list", methods=["POST"])
+def slash_get_list():
+    if verify(request):
+        return {
+            "response_type": "in_channel",
+            "blocks": generate_demo_order_blocks(),
+        }
+    else:
+        return {
+            "response_type": "in_channel",
+            "text": "An error occured with this request"
+        }
+
+
+def post_scheduled_demo_order_msg(event, context):
+    if verify_is_valid_demo_day():
+        sc.chat_postMessage(
+            channel="C01GK7J30A3",  # update to accept a custom channel id
```

Comment:
I'm not sure about the sensitivity of this id, but I think the procurify specific data should be squashed out before this is merged

Reply:
Ah yeah @neilpower, I forgot to update to use this env variable I set here!! Good catch https://github.com/procurify/presentation-order/pull/1/files#diff-568470d013cd12e4f388206520da39ab9a4e4c3c6b95846cbc281abc1ba3c959R15